apps/kitchen-sink/src/components/common/Loaders/Loader.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./Loader.scss";
+
+const Loader = () => {
```

Comment:
Why not to use loader from the library?
https://style.monday.com/?path=/story/components-loader--sizes